sdk/tests/deqp/framework/common/tcuFloat.js

```diff
@@ -415,8 +415,7 @@ tcuFloat.deFloat.prototype.getBitRange = function(begin, end) {
  * @return {number}
  */
 tcuFloat.deFloat.prototype.bits = function() {
-    if (typeof this.bitValue === 'undefined')
-        this.bitValue = deMath.arrayToNumber(this.array());
+    this.bitValue = deMath.arrayToNumber(this.array());
```

Comment:
Does this affect perf of running texture* tests?  We found out that this deFloat class is the main bottleneck (took like 95%+ of running time) in these tests and probably other tests.

If it does, can we implement a dirty bit? So we don't have to recompute each time.